app/controllers/application_controller.rb

```diff
@@ -61,13 +62,15 @@ def deny_access
     (User.current.logged? || request.xhr?) ? render_403 : require_login
   end
 
+  def require_ssl?
+    SETTINGS[:require_ssl]
+  end
+
   def require_ssl
```

Comment:
That's `require_ssl?` (trailing question mark), while my question is about this specific line, an existing filter called `require_ssl`.